packages/storybook/.storybook/main.ts

```diff
@@ -0,0 +1,33 @@
+import MiniCssExtractPlugin from "mini-css-extract-plugin";
+import { VanillaExtractPlugin } from "@vanilla-extract/webpack-plugin";
+// @ts-expect-error
+import * as path from "path";
+import type { StorybookConfig } from "@storybook/react-webpack5";
+
+const config: StorybookConfig = {
+  stories: [{ directory: "../stories", files: "**/*.stories.*" }],
+  addons: ["@storybook/addon-links", "@storybook/addon-essentials", "storybook-addon-themes"],
+  framework: {
+    name: "@storybook/react-webpack5",
+    options: {},
+  },
+  webpackFinal: (baseConfig) => {
+    const { module: { rules = [] } = {}, plugins = [] } = baseConfig;
+
+    // by default, the rules for jsx|tsx files are including the whole monorepo root,
```

Comment:
is this still true? Does it make sense to check if we can remove these lines?

Reply:
I haven't seen anything specific to the webpack config in the changelog, so I'll keep it for now